persistentvolume/persistentvolume.go

```diff
@@ -62,3 +62,12 @@ func (s *Service) Delete(ctx context.Context, name string) error {
 
 	return nil
 }
+
+// DeleteCollection deletes all persistentVolumes.
```

Comment:
I think it's better to try to make interfaces of all services similar.
Currently, the pod service's  DeleteCollection only accepts metav1.ListOptions. So, let's add ListOptions to other service's DeleteCollection.

Reply:
Ok, I fixed it.

Reply:
I know this commit is failed.
I fixed it later.